Tribler/Core/Modules/restapi/display_endpoint.py

```diff
@@ -107,6 +107,9 @@ def render_GET(self, request):
         :param request: the HTTP GET request which specifies the focus node and optionally the neighbor level
         :return: the node data formatted in JSON
         """
+
```

Comment:
Please remove the above newline and add a comment explaining why this header is necessary.